mathics/builtin/comparison.py

```diff
@@ -466,6 +441,39 @@ def apply(self, items, evaluation):
             return Expression("And", *groups)
 
 
+def do_cmp_cplx(x, y) -> Optional[int]:
+    if isinstance(y, Complex):
+        x, y = y, x
+    if isinstance(x, Complex):
+        if isinstance(y, Complex):
+            c = do_cmp(x.real, y.real)
+            if c is None:
+                return
+            if c != 0:
+                return False
+            c = do_cmp(x.imag, y.imag)
```

Comment:
Notice that _this_ `do_cmp` follows how `cmp` is traditionally used.

Reply:
copy that!